File: pgm/ABC_Kotlin/ABC_429_A.kt
// import kotlin.math.*

const val OK: String = "OK"
const val Too: String = "Too Many Requests"
const val zero: Int = 0
const val space: String = " "

fun main() {
    val (N: Int, M: Int) = readln().split(space).map { it.toInt() }
    for (i in zero until N) {
        if (i < M) {
            println(OK)
        } else {
            println(Too)
        }
    }
    kotlin.system.exitProcess(0)
}

File: pgm/ABC_C#/ABC_429_A.cs
using System;
// using System.Collections.Generic;
// using System.Globalization;
// using System.IO;
// using System.Linq;
// using System.Text;
// using System.Text.RegularExpressions;

class Program
{
    static void Main()
    {
        string[] arr = Console.ReadLine().Split(' ');
        const string OK = "OK";
        const string Too = "Too Many Requests";
        const int one = 1;
        const int zero = 0;
        int N = int.Parse(arr[zero]);
        int M = int.Parse(arr[one]);
        for (int i = one; i <= N; i++)
        {
            Console.WriteLine(i <= M ? OK : Too);
        }
        Environment.Exit(0);
    }
}